extensions/amp-ad/0.1/test/test-amp-ad-ui.js

```diff
@@ -52,6 +54,26 @@ describes.realWin('amp-ad-ui handler', {
       expect(attemptCollapseSpy).to.not.be.called;
     });
 
+    it('should force collapse ad inside flying carpet', function* () {
```

Comment:
Can we add another test, that tests the collapse will not happen if the ad is not the only child of flying carpet, or not the direct child of flying carpet.